### ISSUE TITLE
multiple video tracking

### ISSUE THREAD

**Issue by JXXCgit** [DeepLabCut/DeepLabCut#528]
Hi, Dear all, 
Sooorry to bother again.  i have two videos and these two videos are almost mirror  and these two videos are record almost same time (may be 1s-3s gap) and the only difference might be the lightness(but not so strong different). My animals may walk within these two video at different time. 
if i want to analyze these two videos meanwhile, should i also do the label for both, or just only one video is enough for further evaluation? 
Greetings, 
Shauyin

**Comment by MMathisLab** [comment]
We please ask that you do not post usage questions on GitHub. Please see our community resources here: https://github.com/AlexEMG/DeepLabCut#community-support-developers--help. 

We also highly recommend looking over the papers, and the docs about how you want to always make a network using as diverse as data as possible: https://github.com/AlexEMG/DeepLabCut/blob/master/docs/functionDetails.md#c-data-selection-extract-frames

> CRITICAL: A good training dataset should consist of a sufficient number of frames that capture the full breadth of the behavior. This implies to select the frames from different (behavioral) sessions, different lighting and different animals, if those vary substantially (to train an invariant, robust feature detector). Thus, a good training dataset should reflect the diversity of the behavior with respect to postures, luminance conditions, background conditions, animal identities,etc. of the data that will be analyzed. For the behaviors we have tested so far, a data set of 100−200 frames gave good results [Mathis et al, 2018]. However, depending on the required accuracy and the nature of the scene statistics, more or less frames might be necessary to create the training data set.
